src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
```

Comment:
The 'color' field is of type 'String', but since there is an enum 'Color', you should use the enum type instead to restrict the possible values and make the code more type-safe.